### ISSUE TITLE
Exception thrown using MeanSquaredError loss

### ISSUE THREAD

**Issue by dje-dev** [SciSharp/TensorFlow.NET#728]
With a Keras model, model.compile works well when loss is CategoricalCrossentropy. However if instead other losses are used, for example:
```
                loss: keras.losses.MeanSquaredError(),
```
then an Exception is thrown (such as "key SquaredDifference not present").

Is there a way to make the MeanSquaredError work, or perhaps this is not yet currently supported?
```
Unhandled exception. System.Collections.Generic.KeyNotFoundException: The given key 'SquaredDifference' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Tensorflow.Eager.EagerRunner.<>c__DisplayClass5_0.<GetGradientFunction>b__0(Tensor[] output_grads, Int64[] unneeded_gradients)
   at Tensorflow.Gradients.Tape.ComputeGradient(Int64[] target_tensor_ids, Int64[] source_tensor_ids, UnorderedMap`2 sources_that_are_targets, Tensor[] output_gradients)
   at Tensorflow.Gradients.GradientTape.gradient(Tensor target, IEnumerable`1 sources)
   at Tensorflow.Keras.Engine.Model._minimize(GradientTape tape, OptimizerV2 optimizer, Tensor loss, List`1 trainable_variables)
   at Tensorflow.Keras.Engine.Model.train_step(Tensor x, Tensor y)
   at Tensorflow.Keras.Engine.Model.step_function(OwnedIterator iterator)
   at Tensorflow.Keras.Engine.Model.FitInternal(Int32 epochs)
   at Tensorflow.Keras.Engine.Model.fit(IDatasetV2 dataset, IDatasetV2 validation_data, Int32 batch_size, Int32 epochs, Int32 verbose, Single validation_split, Boolean shuffle, Int32 initial_epoch, Int32 max_queue_size, Int32 workers, Boolean use_multiprocessing)
   at TensorFlowNET.Examples.CIFAR10_CNN.Train() in C:\dev\CeresDev\src\Ceres.Samples\Tensorflow\ToyResNet.cs:line 99
   at TensorFlowNET.Examples.CIFAR10_CNN.Run() in C:\dev\CeresDev\src\Ceres.Samples\Tensorflow\ToyResNet.cs:line 31
   at Ceres.Samples.Program.Main(String[] args) in C:\dev\CeresDev\src\Ceres.Samples\Program.cs:line 443
```

**Comment by Oceania2018** [comment]
Should have been fixed for [v0.4.0](https://www.nuget.org/packages/TensorFlow.Keras/).
